testing/run_ap_with_constraint.py

```diff
@@ -18,6 +18,8 @@ def command_parser():
     default_test_cases_path = os.path.join(os.getcwd(), "tests")
     parser.add_argument("-test_cases_path", default=default_test_cases_path, type=str, help="path of to intermediate files of the testcases")
     parser.add_argument("-chan_width", default=100, type=int, help="largest chan width required for tests in this test suite")
+    parser.add_argument("-run_simulated_annealing", action="store_true", help="test simulated annealing")
```

Comment:
Same comment as above. Try to make this a bit more clear (help message + maybe the argument itself).